pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -169,16 +171,16 @@ type ConfigProducerArgs struct {
 	InterPodAffinityArgs *config.InterPodAffinityArgs
 }
 
-// ConfigProducer returns the set of plugins and their configuration for a
+// configProducer returns the set of plugins and their configuration for a
```

Comment:
Update this comment, we modified the pass-in args in place now.

And rename `configProducer` to sth. else? like just simply `configure` or `configFn`?

Reply:
Updated comment, but I prefer to leave the name as it's not too far off. Hopefully this file doesn't live for much longer.